zuul.d/jobs.yaml

```diff
@@ -79,6 +79,16 @@
       - ^molecule-requirements.txt
       - ^ci_framework/roles/edpm_deploy/(?!meta).*
       - ^ci/playbooks/molecule.*
+- job:
+    name: cifmw-molecule-hive
+    parent: cifmw-molecule-base
+    vars:
+      TEST_RUN: hive
```

Comment:
Where is your molecule folder in this role? It is generated by `$ make new_role ROLENAME=wonderful_role` no?